### ISSUE TITLE
[Bug]: Local Hosting - Install/ Setup documentation needs fixing very badly. 

### ISSUE THREAD

**Issue by radhakrisri** [QuivrHQ/quivr#2408]
### What happened?

TLDR version: The setup instructions here: https://docs.quivr.app/developers/contribution/install and here: https://github.com/QuivrHQ/quivr?tab=readme-ov-file#60-seconds-installation- do not work. They need updating very badly.

Summary of multiple trials for the past week:

1. Installed Supabase CLI
2. Followed instructions here: https://github.com/QuivrHQ/quivr?tab=readme-ov-file#60-seconds-installation- to the dot. 
3. Opened Supabase studio to notice that no tables were created. 
4. Had to run `supabase migration up` to have the tables setup. 
5. Created a separate user account in supabase authentication section. 
6. Opened http://localhost:3000 and tried logging in. Encountered this error. 

![image](https://github.com/QuivrHQ/quivr/assets/125212515/2f61e105-29f5-4816-90b0-3cd028f3be04)

7. Noticed in users table that column "onboarded" is set to "False" for the user I created. 
8. Have scoured through the github issues and documentation much, but haven't found enough info. 

All in all, the documentation, and probably the setup procedure, needs to be fixed very badly.

**Comment by radhakrisri** [comment]
@StanGirard : Seeking your guidance here!

**Comment by radhakrisri** [comment]
Looks like there's one more issue reported here: https://github.com/QuivrHQ/quivr/issues/2354

**Comment by powderblock** [comment]
Hello, what operating system are you running?

Are you running docker / supabase locally?

**Comment by StanGirard** [comment]
Hey my friend, can you try to git pull and rerun the `docker compose up` ?

Or else, follow this procedure if on your own vps: https://docs.quivr.app/deployment/digital_ocean

It should fix your problems :)

**Comment by concaption** [comment]
Did any one find a solution? Every time I try to run it it gets into some error. It is also saying that it is opensource and unpaid user will have limited access at the same time.

I think the issue is that I do not see the supabase container in the dockerfile and would need to be installed via some shit named scoop. 

Do let me know if someone has resolved the issue. Also I was not able to figure out what will be the admin credentials.

Do anyone know something similar that uses ollama but is better documented.
